ome2024-ngff-challenge/src/About.svelte

```diff
@@ -75,10 +75,24 @@
     a federated bioimage data system based on OME-Zarr, the largest one we know of.
     We hope future challenges will continue to push the state-of-the-art forward.
   </p>
+
+  <h2>Funding</h2>
+  <p>
+    Coordination of the challenge has been supported by the <a href="https://www.nfdi.de/">German National Research Data
+    Initiative (NFDI)</a> and the <a href="https://chanzuckerberg.com/">Chan Zuckerberg Initiative</a>:
+  </p>
+
+  <table>
+    <tr>
+      <td><img src="nfdi_rgb_Wortmarke_Zusatz_quer.png" width="45%"/></td>
+      <td><img src="czi-logo-chan-zuckerberg-initiative-logo.png" width="45%"/></td>
```

Comment:
I was going to say that this won't work when deployed on github, and suggest that you need to do it as in Nav.svelte:

```
import omeMainNav from "/ome-main-nav.svg";
...
<img alt="OME Logo" src={omeMainNav} />
```

But I just pushed this branch to my 'main' branch so it's deployed at https://will-moore.github.io/ome2024-ngff-challenge/about/ and it's fine!
The approach I used for Nav is *possibly* a bit more robust since it generates an absolute URL, but actually this is working fine so 👍

Reply:
Looks to be working. Thanks.